go.mod

```diff
@@ -174,11 +174,11 @@ require (
 	k8s.io/kube-aggregator v0.30.1
 	k8s.io/kubectl v0.30.1
 	k8s.io/kubernetes v1.30.1
-	k8s.io/utils v0.0.0-20240102154912-e7106e64919e
+	k8s.io/utils v0.0.0-20240502163921-fe8a2dddb1d0
 	oras.land/oras-go/v2 v2.4.0
 	sigs.k8s.io/aws-iam-authenticator v0.6.17
 	sigs.k8s.io/cluster-api v1.7.3
-	sigs.k8s.io/controller-runtime v0.18.4
+	sigs.k8s.io/controller-runtime v0.18.5
```

Comment:
I'm not sure why this was bumped here. Near as I can tell, none of the other 3 libraries that were bumped use this library, so I think we should be able to bump the other ones without also bumping this dependency.

This being said, our use of this (as a go dependency) seems pretty limited, so if it's an automatic bump that would be hard to avoid I don't have an issue with keeping it as-is.

Reply:
Resolving, this is used in lasso and pulled in by the more recent changes.